adafruit_ov7670.py

```diff
@@ -611,16 +619,16 @@ def height(self):
         return 480 >> self._size
 
     @property
-    def colorspace(self):
+    def colorspace(self) -> Optional[int]:
```

Comment:
I think colorspace() return can be `int` without `Optional`.

I saw that `self._colorspace` does start as None, but immediately after being initialized it gets set to `OV7670_COLOR_RGB` and from then on seems to assume that the value will be either `OV7670_COLOR_RGB` or `OV7670_COLOR_YUV`